src/HealthGPS.Console/model_parser.cpp

```diff
@@ -424,9 +410,41 @@ load_kevinhall_risk_model_definition(const poco::json &opt, const host::Configur
     age_mean_height.emplace(hgps::core::Gender::male, std::move(male_height));
     age_mean_height.emplace(hgps::core::Gender::female, std::move(female_height));
 
+    // Weight quantiles.
+    const auto weight_quantiles_table_F = load_datatable_from_csv(
+        host::get_file_info(opt["WeightQuantiles"]["Female"], config.root_path));
+    const auto weight_quantiles_table_M = load_datatable_from_csv(
+        host::get_file_info(opt["WeightQuantiles"]["Male"], config.root_path));
+    std::unordered_map<hgps::core::Gender, std::vector<double>> weight_quantiles = {
+        {hgps::core::Gender::female, {}}, {hgps::core::Gender::male, {}}};
+    weight_quantiles[hgps::core::Gender::female].reserve(weight_quantiles_table_F.num_rows());
+    weight_quantiles[hgps::core::Gender::male].reserve(weight_quantiles_table_M.num_rows());
+    for (size_t j = 0; j < weight_quantiles_table_F.num_rows(); j++) {
+        weight_quantiles[hgps::core::Gender::female].push_back(
+            std::any_cast<double>(weight_quantiles_table_F.column(0).value(j)));
+    }
+    for (size_t j = 0; j < weight_quantiles_table_M.num_rows(); j++) {
+        weight_quantiles[hgps::core::Gender::male].push_back(
+            std::any_cast<double>(weight_quantiles_table_M.column(0).value(j)));
+    }
+    for (auto &[sex, quantiles] : weight_quantiles) {
+        std::sort(quantiles.begin(), quantiles.end());
+    }
```

Comment:
Why sorting them? Aren't the elements picked at random?

Reply:
The idea here is that we want to be able to pick out one at a certain percentile, for example the 90th percentile (here 0.9) at the bottom of the top 10% of values. 

Because we are taking this percentile and converting it to a percentile to index another list, we want that other list to have similar ordering -- i.e. sorted in ascending order.